var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -46,19 +47,26 @@ class PyCelery(PythonPackage):
     # 'cosmosdbsql',
     # 's3',
 
+    depends_on('python@3.7:', type=('build', 'run'), when="@5.2.3:")
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
 
     depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
In the latest version it requires `py-setuptools@59.1.1:59.6`

Reply:
I'll add this in shortly.